cli/azd/internal/repository/app_init.go

```diff
@@ -551,6 +619,44 @@ func (i *Initializer) prjConfigFromDetect(
 			dbNames[database] = db.Name
 		}
 
+		for _, azureDepPair := range detect.AzureDeps {
+			azureDep := azureDepPair.first
+			_authType, err := i.chooseAuthTypeByPrompt(ctx, azureDep.ResourceDisplay())
+			if err != nil {
+				return config, err
+			}
+			authType := "connectionString"
+			if _authType == scaffold.AuthType_PASSWORD {
+				authType = "password"
+			} else if _authType == scaffold.AuthType_TOKEN_CREDENTIAL {
+				authType = "managedIdentity"
+			}
+			switch azureDep.(type) {
+			case appdetect.AzureDepServiceBus:
+				azureDepServiceBus := azureDep.(appdetect.AzureDepServiceBus)
+				config.Resources["servicebus"] = &project.ResourceConfig{
+					Type: project.ResourceTypeMessagingServiceBus,
+					Props: project.ServiceBusProps{
+						Queues:   azureDepServiceBus.Queues,
+						IsJms:    azureDepServiceBus.IsJms,
+						AuthType: authType,
+					},
+				}
+			case appdetect.AzureDepEventHubs:
```

Comment:
Seems that never construct `ResourceTypeMessagingKafka` here. We can decide by `AzureEventHubs.UseKafka`

Reply:
Yes, we could do that.